dataland-frontend/src/components/forms/CreateEUTaxonomy.vue

```diff
@@ -206,7 +207,7 @@ export default {
   methods: {
     async getCompanyIDs(){
       try {
-        const companyList = await companyStore.perform("")
+        const companyList = await getCompaniesWrapper.perform("", "", true)
         this.idList = companyList.data.map(element => element.companyId)
       } catch(error) {
         this.idList = [0]
```

Comment:
Könnte das nicht schief gehen, falls wir irgendwann keinen sauberen Counter mehr für companyId haben (der bei 1 beginnt), und es daher auch die companyId 0 geben kann?

Reply:
Könnte man im error Fall auch eine leere Liste zuweisen?

Reply:
Ok.
Aber aus Interesse: Warum kann man im catch nicht this.idList = [] machen?  Also leere Liste returnen.